tests/testthat/test_eml.R

```diff
@@ -481,7 +481,8 @@ test_that('eml_nsf_to_project fails gracefully', {
 test_that('eml_nsf_to_project parses two-word last names correctly', {
   proj <- eml_nsf_to_project("1822406", eml_version = "2.2")
 
-  expect_equal(proj$personnel$individualName$givenName, "Maria Val Martin")
+  expect_equal(proj$personnel$individualName$givenName, "Maria")
```

Comment:
realized this fails because there is more than one personnel in a given project!